src/main/java/org/fiware/iam/tir/rest/IShareMapper.java

```diff
@@ -0,0 +1,45 @@
+package org.fiware.iam.tir.rest;
+
+import lombok.extern.slf4j.Slf4j;
+import org.fiware.iam.satellite.model.AdherenceVO;
+import org.fiware.iam.satellite.model.CertificateVO;
+import org.fiware.iam.satellite.model.PartyVO;
+import org.fiware.iam.tir.auth.JWTService;
+import org.fiware.iam.tir.configuration.Party;
+
+import javax.inject.Singleton;
+import java.security.cert.CertificateEncodingException;
+import java.security.cert.X509Certificate;
+import java.util.Base64;
+import java.util.List;
+
+@Singleton
+@Slf4j
```

Comment:
Missing doc

Reply:
Why did you decide againts mapstruct here? Just out of curiosity, at least partyToPartyVO should be possible